web/cso_area/add_new_activity.php

```diff
@@ -68,80 +66,85 @@
-                                          <div class="col-sm-10">
+                                
+                                    
+                                      <div class="form-group ">
+                                          <label for="colFormLabel" class="cotrol-label">Expected Output</label>
+                                          
                                             <input type="text" class="form-control" id="colFormLabel" placeholder="Expected Output">
                                           </div>
-                                        </div>
+                                       
 
-                                      </div>
-                                    </div>
+                                      
+                                    
 
                                     <div class="row">
-                                      <div class="col-md-6">
-                                        <div class="form-group row">
-                                            <label for="colFormLabel" class="col-sm-4 col-form-label">Start Date</label>
-                                            <div class="col-sm-10">
-                                              <input type="text" class="form-control" id="colFormLabel" placeholder="start date">
+                                     
+                                        <div class=" col-sm-6 ">
+                                            <label for="colFormLabel" class=" control-label">Start Date</label>
+                                            <div class="input-group datetimepicker">
+                                              <input type="text" class="form-control " id="colFormLabel" placeholder="start date">
+                                              <span class="input-group-addon">
+                        <span class="glyphicon glyphicon-calendar"></span>
+                    </span>
                                             </div>
-                                          </div>
+                                          
 
                                         </div>
 
-                                        <div class="col-md-6">
-                                          <div class="form-group row">
-                                              <label for="colFormLabel" class="col-sm-4 col-form-label">End Date</label>
-                                              <div class="col-sm-10">
-                                                <input type="text" class="form-control" id="colFormLabel" placeholder="end date">
-                                              </div>
+                                       
+                                          <div class=" col-sm-6">
+                                              <label for="colFormLabel" class="control-label">End Date</label>
+                                              <div class="input-group datetimepicker">
```

Comment:
@sempermort date picker not working

Reply:
there might be a broblem with js extension s